.circleci/config.yml

```diff
@@ -759,132 +618,8 @@ jobs:
-      GOARCH: "386"
-      PLATFORM: linux-32bit
-
-  operator-build-windows-386:
-    <<: *go_build
-    environment:
-      GOOS: windows
-      GOARCH: "386"
-      PLATFORM: windows-32bit
-      EXESUFFIX: ".exe"
-
-  operator-build-windows-amd64:
-    <<: *go_build
-    environment:
-      GOOS: windows
-      GOARCH: amd64
-      PLATFORM: windows-64bit
-      EXESUFFIX: ".exe"
-
-  operator-build-darwin-386:
-    <<: *go_build
-    environment:
-      GOOS: darwin
-      GOARCH: "386"
-      PLATFORM: mac-32bit
-
-  operator-build-darwin-amd64:
-    <<: *go_build
-    environment:
-      GOOS: darwin
-      GOARCH: amd64
-      PLATFORM: mac-64bit
-
-  operator-build-linux-arm:
-    <<: *go_build
-    environment:
-      GOOS: linux
-      GOARCH: arm
-      PLATFORM: linux-arm32bit
-
-  operator-build-linux-arm64:
-    <<: *go_build
-    environment:
-      GOOS: linux
-      GOARCH: arm64
-      PLATFORM: linux-arm64bit
-
-  operator-test:
-    <<: *go_build
-    steps:
-      - checkout
-      - <<: *test_go_executable
-
-  operator-publish-github-release:
-    docker:
-      - image: cibuilds/github:0.10
-    steps:
-      - attach_workspace:
-          at: .
-      - run:
-          name: "Create a Draft Release on GitHub"
-          command: |
-            VERSION="$CIRCLE_TAG"
-            BASEDIR=`pwd`
-            mkdir "${BASEDIR}/archives"
-            for p in `ls dist` ; do
-              cd "$BASEDIR/dist/$p"
-              if [[ $p == windows* ]] ; then
-                zip -q "${BASEDIR}/archives/syndesis-${VERSION}-$p.zip" *
-              else
-                tar -zcf "${BASEDIR}/archives/syndesis-${VERSION}-$p.tgz" *
-              fi
-            done
-            cd ${BASEDIR}
-            ghr -u ${GITHUB_USER} -t ${GITHUB_TOKEN} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -replace ${VERSION} "${BASEDIR}/archives"
 workflows:
   version: 2
-  release:
-    jobs:
-      - operator-build-linux-386:
-          <<: *release_filters
-      - operator-build-linux-amd64:
-          <<: *release_filters
-      - operator-build-darwin-386:
-          <<: *release_filters
-      - operator-build-darwin-amd64:
-          <<: *release_filters
-      - operator-build-windows-386:
-          <<: *release_filters
-      - operator-build-windows-amd64:
-          <<: *release_filters
-      - operator-build-linux-arm:
-          <<: *release_filters
-      - operator-build-linux-arm64:
-          <<: *release_filters
-      - operator-test:
-          <<: *release_filters
-      - operator-publish-github-release:
-          <<: *release_filters
-          requires:
-            - operator-build-linux-386
-            - operator-build-linux-amd64
-            - operator-build-darwin-386
-            - operator-build-darwin-amd64
-            - operator-build-windows-386
-            - operator-build-windows-amd64
-            - operator-build-linux-arm
-            - operator-build-linux-arm64
-            - operator-test
-
```

Comment:
But this here doesn't hurt ...

Reply:
It does as it's not using the `syndesis` shell script, the `syndesis` shell script is **the** way of running any build. If there are special cases for the operator those need to be added to the `syndesis` shell script, otherwise we won't be portable accross CI environments and contributors won't be able to run any check performed by CI locally.

Reply:
you are right ... but this very snippet is not the cause the our issues. By removing it now, you leave my without a place where the binaries are uploaded, `syndesis crc --install` and `syndesis minishift --install` won't work, very likely.

Can we find a compromise here? I can work migrating them into the release script asap

Reply:
@lgarciaaco was the author of that in search of a compromise, or a solution that would be in line with the workflows and tools already in place. Could I also be allowed this. As [I've mentioned](https://groups.google.com/forum/#!topic/syndesis/O7Ih9Oi6wZc) in the Syndesis Google group, you're free to add this support to the `syndesis` shell script.

Reply:
@zregvart when these bits break the release remove it. If this is not the case I suggest to keep 'em and to not break the `syndesis crc --install` procedure until we have a better way (aligned with the release scripts).

AFAIU @lgarciaaco already volunteered to consolidate this with the release scripts. After that we can safely remove this.